tests/unit/test_discovery_client.py

```diff
@@ -35,6 +42,11 @@
     ui_file_paths=[],
 )
 
+_MOCK_KEY_FILE_CONTENT = json.dumps({"SecurePort": "", "InsecurePort": _TEST_SERVICE_PORT})
+_MOCK_REGISTRATION_FILE_CONTENT = json.dumps(
+    {"discovery": {"path": "Discovery/NationalInstruments.MeasurementLink.DiscoveryService.exe"}}
+)
```

Comment:
I think moving the `json.dumps` to the fixture would slightly simplify the test cases. `_MOCK_REGISTRATION_FILE_CONTENT` would be a dict and the test case wouldn't have to call `json.loads`.

Reply:
Moved `json.dumps` to fixture